projects/boringssl/build.sh

```diff
@@ -48,3 +46,20 @@ for F in $fuzzerFiles; do
     zip -j $OUT/${fuzzerName}_seed_corpus.zip $SRC/boringssl/fuzz/${fuzzerName}_corpus/*
   fi
 done
+
+rm -rf genfiles && mkdir genfiles && $SRC/LPM/external.protobuf/bin/protoc asn1_pdu.proto --cpp_out=genfiles --proto_path=$SRC/fuzzing/proto/asn1-pdu-proto/
```

Comment:
LPM fuzz targets do not work with MSan, we'll have to have a check here and either bail out or put all LPM commands inside the `if ... then` body